src/mon/Monitor.cc

```diff
@@ -2271,6 +2271,120 @@ void Monitor::do_health_to_clog(bool force)
   health_status_cache.summary = summary;
 }
 
+namespace {
```

Comment:
To give the functions inside internal linkage.
Some more info:
http://stackoverflow.com/questions/4977252/why-unnamed-namespace-is-a-superior-alternative-to-static